frontend/components/bc/BcButton.vue

```diff
@@ -1,5 +1,7 @@
 <script lang="ts" setup>
-const props = defineProps<{
+const {
+  isAriaDisabled, isDisabled, type = 'button',
```

Comment:
np: this should be taken care of `eslint` 
🫣 I think I did not manage before to make this work reliably
```suggestion
  isAriaDisabled,
  isDisabled,
  type = 'button',
```

Reply:
It seems that none of the `eslint` configurations we use (or `eslint` provides) are meant to take care of this.
`@stylistic/object-property-newline` doesn't mention destructuring 
`@stylistic/object-curly-newline` deals only with the newline before and after curly braces, not between properties.

I added a `newline-destructuring` plugin and it works well. The plugin doesn't have `ts` and was updated a year ago, but it has no known vulnerabilities and is rather widely used.